lib/rubocop/cop/rspec/expectation_target_method.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      # Checks ExpectationTarget method.
```

Comment:
imo) The word “Checks” seems to be unclear as to whether it is or is not an offense as a result of the check, although this is a formality that exists in many other cops. I think it would be better if it is explained when it is a violation, or what kind of format is recommended by the cop.

Reply:
I have come up with the following explanation, what do you think of this explanation?
```
Checks if the ExpectationTarget methods `.to`, `not_to` or `to_not` are used.
```

Reply:
How about we change the following?

```
Checks if the RSpec::Expectations::ExpectationTarget methods `.to`, `not_to` or `to_not` are used.
```

Reply:
Thanks for the suggestion! I tried the fix but had problems :cry: 

First, I tried to write it in one line, but was registered an offense by `Layout/LineLength`.

```
lib/rubocop/cop/rspec/missing_expectation_target_method.rb:6:81: C: Layout/LineLength: Line is too long. [106/80]
      # Checks if the RSpec::Expectations::ExpectationTarget methods `.to`, `not_to` or `to_not` are used.
                                                                                ^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Next, the test failed when I included newlines.

```
  1) config/default.yml does not have newlines in cop descriptions
     Failure/Error: expect(value).not_to include("\n")
       expected "Checks if the RSpec::Expectations::ExpectationTarget methods `.to`,\n`not_to` or `to_not` are used." not to include "\n"
     # ./spec/project/default_config_spec.rb:80:in `block (3 levels) in <top (required)>'
     # ./spec/project/default_config_spec.rb:79:in `each'
     # ./spec/project/default_config_spec.rb:79:in `block (2 levels) in <top (required)>'
```

In the `build_config` task, the first line of the document is set to `description` in default.yml, but it seems that newlines are not allowed.

I tried to simplify the first line and write the details separately as follows.

```
Checks if `.to`, `not_to` or `to_not` are used.

The RSpec::Expectations::ExpectationTarget must use `to`, `not_to` or
`to_not` to run. Therefore, this cop checks if other methods are used.
```